vault/hooks/health_check

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+rc=0
+
+protocol=http{{#unless cfg.listener.tls_disable}}s{{/unless}}
+
+status=$({{pkgPathFor "core/curl"}}/bin/curl \
```

Comment:
since `core/curl` is a runtime dep, it should be in your `PATH` so you shouldn't have to `pkgPathFor` it at the time your `health_check` runs.

Reply:
You're absolutely right. I've pushed a commit to change it.